platform/jobs/edxPlatformQualityPr.groovy

```diff
@@ -166,43 +139,40 @@ jobConfigs.each { jobConfig ->
                 }
             }
         }
-
-        configure GHPRB_WHITELIST_BRANCH(jobConfig.whitelistBranchRegex)
-
         wrappers {
             timeout {
                 absolute(90)
             }
             timestamps()
             colorizeOutput()
-            buildName('#\${BUILD_NUMBER}: Quality Tests')
```

Comment:
Until we do https://openedx.atlassian.net/browse/TE-2333, I think we will need to keep the sshAgent part for private jobs